yarn.lock

```diff
@@ -4290,6 +4296,13 @@ promise@^7.1.1:
   dependencies:
     asap "~2.0.3"
 
+promisify-node:
+  version "0.4.0"
+  resolved "https://registry.yarnpkg.com/promisify-node/-/promisify-node-0.4.0.tgz#32803874ec411784e4786c339902a87a179a469c"
+  dependencies:
+    nodegit-promise "~4.0.0"
+    object-assign "^4.0.1"
+
 prr@~0.0.0:
   version "0.0.0"
   resolved "https://registry.yarnpkg.com/prr/-/prr-0.0.0.tgz#1a84b85908325501411853d0081ee3fa86e2926a"
```

Comment:
when you remove a project, you need to re-instantiate the lock. we should be seeing red here, for portmapper